### ISSUE TITLE
Reproducing SoTA results for QM9

### ISSUE THREAD

**Issue by XJTUNR** [torchmd/torchmd-net#75]
Hello Author,

I noticed that your powerful ET model achieves SoTA results on 5 tasks in QM9, and I would like to reproduce the results. 

Unfortunately, when I adopted the QM9 template from your repo and turn on/off Atomref, Standardization and change the output modules accordingly, the reproduced results (after early stopping) are still much lower than those reported in the paper. 

For example, the result of dipole moment MAE is 0.01 vs 0.0002, the result of electronic spatial extent is 0.035 vs 0.015.

I'm curious whether you have adjusted the hyper-parameters for each property. Can you share the hyper-parameters that can reproduce the results with me?

Thanks for your help!

**Comment by lsnty5190** [comment]
Hi! I have the same problems too. I wonder if we should adjust the random seed or other hyparams like num layers and learning rate?

**Comment by PhilippThoelke** [comment]
Hi, thanks for your interest! This discrepancy unfortunately comes from a bug in the model architecture, which is fixed now (you seem to be using the fixed version, see #58). We have already updated the the paper on arxiv but the new version is not yet published (will happen next week). The bug did not affect most of our results (especially MD17 and ANI1 remained the same or got better), however, accuracy on few targets in the QM9 dataset got worse after the bug was fixed. Sorry for the inconvenience!

**Comment by XJTUNR** [comment]
Let me finally understand that whether you used the same hyperparameters to train all the properties in theQM9 dataset? 

Would it be convenient to share the table after updating?

:) Thanks for your reply.

**Comment by giadefa** [comment]
These are the final numbers which you should be able to obtain:
https://openreview.net/pdf?id=zNHzqZ9wrRB

**Comment by XJTUNR** [comment]
Yep. 

Thanks for your information! :)
